File: HrNewPortal.Models/PollOption.cs
using System;

namespace HrNewsPortal.Models
{
    public class PollOption
    {
        /// <summary>
        /// The item's unique identifier.
        /// </summary>
        public int Id { get; set; }

        /// <summary>
        /// The user name of the item's author.
        /// </summary>
        public string By { get; set; }

        /// <summary>
        /// Creation UTC date of the item.
        /// </summary>
        public DateTime? Time { get; set; }

        /// <summary>
        /// The comment story or poll text.
        /// </summary>
        public string Text { get; set; }

        /// <summary>
        /// The poll option's associated poll identifer.
        /// </summary>
        public int PollId { get; set; }

        /// <summary>
        /// The poll option's associated poll.
        /// </summary>
        public Poll Poll { get; set; }
        
        /// <summary>
        /// The story's score, or the votes
        /// for a poll option.
        /// </summary>
        public int Score { get; set; }
    }
}

File: HrNewsPortal.Portal/ClientApp/src/app/models/poll-option.ts
import { Poll } from "./poll";

export class PollOption {
  // The item's unique identifier.
  public id: number;

  // The user name of the item's author.
  public by: string;

  // Creation UTC date of the item.
  public time?: Date;

  // The comment story or poll text.
  public text: string;

  // The poll option's associated poll identifer.
  public pollId: number;

  // The poll option's associated poll.
  public poll: Poll;

  // The story's score, or the votes
  // for a poll option.
  public score: number;
}
